.github/workflows/deploy-orchard1-to-azure-app-service.yml

```diff
@@ -3,14 +3,22 @@ name: Deploy Orchard 1 to Azure App Service
 concurrency:
   group: AzureDeployApp
 
+permissions:
+  id-token: write
+  contents: read
+
 on:
   workflow_call:
     secrets:
       CHECKOUT_TOKEN:
```

Comment:
This is not new code but a new syntax error: "Missing property "required"." So please add it. Also in the other workflows.

Reply:
Actually, do all of those secrets need to be required? There are probably cases (e.g., for an open source repo) where it's needed.

Reply:
I mean I can add that for sure, but the question is whether it's actually correct or some of them should be required. If yes, then reviewing that and correcting should be in a separate issue.

Reply:
All right then!

Reply:
I've checked all the workflows and there were other secrets with this validation error, but none of them need to be required.

Reply:
Thanks!